settings.gradle.kts

```diff
@@ -6,6 +6,17 @@ pluginManagement {
     }
 }
 
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+    }
+    versionCatalogs {
+        create("ktorLibs") {
```

Comment:
In my opinion, this complicates dependency management

Reply:
This was reverted.